index.html

```diff
@@ -29,25 +38,28 @@ <h2>I am a Software Developer</h2>
         <section class="works">
             <div class="intro">
```

Comment:
Make sure you add the divider below the title in the mobile version.

<img width="356" alt="Screen Shot 2022-07-07 at 22 20 58" src="https://user-images.githubusercontent.com/82426682/177910581-b3016684-0c30-4c55-a9e7-2f9b27cc40ef.png">